### DISCUSSION TITLE
Problem with motion generation when changing the ee_orientation

### DISCUSSION THREAD

**Discussion by Follograph** [NVlabs/curobo#204] [Software Q&A] [answered]
Hi, I'm using motion_gen to generate the trajectory and I want the grasp hand to catch the object cylinder with 90 degrees rotate around the z-axis. That is to say, the origin ee_orientation is [0,1,0,0], and now I change it to [0.70710678,0,0,0.70710678]. That comes to the warning "Plan did not converge to a solution". I have tried changing the orientation of the cylinder itself and the ee_orientation_teleop_goal, but both failed to generate the solution.

However, when I change the ee_orientation to rotate around x or y axis, such as [0.70710678,0.70710678,0,0], it works. What's the problem?

Thanks!

**Comment by balakumar-s** [comment]
What robot are you using? Is it possible to share a minimum python script that we could run to observe the issue?

**Reply by Follograph** [reply, marked as answer]
I'm using a self-defined flexiv robot. Your reply inspired me and I tried the example using franka robot. Then I found when changing to [0.70710678,0,0,0.70710678], the gripper will catch object from below. In my script the object is on the ground so the motion generation failed. The default orientation [0,1,0,0] has the object rotating about the x-axis by 180 degrees but I didn't notice. Thanks!
